changes/gem5-resources.tex

```diff
@@ -0,0 +1,13 @@
+\subsection[gem5 resources]{gem5 resources\footnote{by Mahyar Samani}}
+
+gem5 resources are the set of inputs used to conduct computer systems architecture
+research on known-good configurations using gem5. They include anything from the config
+files used to build a linux kernel that works with gem5 to the config scripts that describe
+the computer system to be simulated. The provided resources have been tested with gem5-20
+and their working status and initial statistics have been documented. They could be used to
+do research with different system configurations. Moreover, some of the provided resources
```

Comment:
Add this statement here or somewhere else you feel appropriate: "We provide gem5 resources to conduct experiments with many popular benchmark suites like SPEC 2006, SPEC 2017, PARSEC, NAS Parallel Benchmarks (NPB), GAP Benhmark Suite (GAPBS) and Linux Kernel." (Also cite all benchmark suites)